src/components/Reactions/hooks/useProcessReactions.tsx

```diff
@@ -9,15 +9,21 @@ import type { ReactionsComparator, ReactionSummary } from '../types';
 type SharedReactionListProps =
   | 'own_reactions'
   | 'reaction_counts'
+  | 'reaction_groups'
   | 'reactionOptions'
   | 'reactions';
 
 type UseProcessReactionsParams = Pick<ReactionsListProps, SharedReactionListProps> & {
   sortReactions?: ReactionsComparator;
 };
 
-export const defaultReactionsSort: ReactionsComparator = (a, b) =>
-  a.reactionType.localeCompare(b.reactionType, 'en');
+export const defaultReactionsSort: ReactionsComparator = (a, b) => {
+  if (a.firstReactionAt && b.firstReactionAt) {
+    return +a.firstReactionAt - +b.firstReactionAt;
+  }
+
+  return a.reactionType.localeCompare(b.reactionType, 'en');
+};
```

Comment:
New default reaction order is chronological, sorted by first time someone reacted. Falls back to alphabetical for older messages without reaction groups.